diff of execution of:
echo 'echo this"has"quotes\'inside\'of"\'it\'"' | tr ';' '\n' | [shell]
--- expected	2024-08-27 19:27:25.180723180 +0200
+++ found	2024-08-27 19:27:25.198723384 +0200
@@ -1,6 +1,6 @@
-thishasquotesinsideof'it'
+thishasquotesinsideofit
 
 ======== stderr =======
 
 ======== return =======
 return value: 0
